src/main/java/com/softdevsix/domain/repositories/ProjectRepository.java

```diff
@@ -2,29 +2,48 @@
 
 import com.softdevsix.domain.entities.project.Project;
 import org.springframework.stereotype.Repository;
-
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
 
 @Repository
 public class ProjectRepository implements IProjectRepository {
```

Comment:
When you use `JpaRepository` interface, you'll see we won't need a concrete for this class and others one.